src/sidebar/components/test/top-bar-test.js

```diff
@@ -1,183 +1,205 @@
+      const wrapper = createTopBar({ auth, onLogout });
+      assert.isFalse(getLoginText(wrapper).exists());
 
-    loginControl.onLogout();
-    assert.called(onLogout);
+      const userMenu = wrapper.find(UserMenu);
+      assert.isTrue(userMenu.exists());
+      assert.include(userMenu.props(), { auth, onLogout });
+    });
   });
 
-  it("checks whether we're using a third-party service", function() {
+  it("checks whether we're using a third-party service", () => {
     createTopBar();
 
     assert.called(fakeIsThirdPartyService);
     assert.alwaysCalledWithExactly(fakeIsThirdPartyService, fakeSettings);
   });
 
-  context('when using a first-party service', function() {
-    it('shows the share page button', function() {
-      let el = createTopBar();
-      // I want the DOM element, not AngularJS's annoying angular.element
-      // wrapper object.
-      el = el[0];
-
-      assert.isNotNull(el.querySelector('[title="Share this page"]'));
+  context('when using a first-party service', () => {
+    it('shows the share page button', () => {
+      const wrapper = createTopBar();
+      assert.isTrue(wrapper.exists('[title="Share this page"]'));
     });
   });
 
-  context('when using a third-party service', function() {
-    beforeEach(function() {
+  context('when using a third-party service', () => {
+    beforeEach(() => {
       fakeIsThirdPartyService.returns(true);
     });
 
-    it("doesn't show the share page button", function() {
-      let el = createTopBar();
-      // I want the DOM element, not AngularJS's annoying angular.element
-      // wrapper object.
-      el = el[0];
-
-      assert.isNull(el.querySelector('[title="Share this page"]'));
+    it("doesn't show the share page button", () => {
+      const wrapper = createTopBar();
+      assert.isFalse(wrapper.exists('[title="Share this page"]'));
     });
   });
 
-  it('displays the share page when "Share this page" is clicked', function() {
+  it('displays the share page when "Share this page" is clicked', () => {
     const onSharePage = sinon.stub();
-    const el = createTopBar({ onSharePage: onSharePage });
-    el.find('[title="Share this page"]').click();
-
+    const wrapper = createTopBar({ onSharePage });
+    wrapper.find('[title="Share this page"]').simulate('click');
     assert.called(onSharePage);
   });
 
-  it('displays the search input and propagates query changes', function() {
-    const onSearch = sinon.stub();
-    const el = createTopBar({
-      searchController: {
-        query: sinon.stub().returns('query'),
-        update: onSearch,
-      },
-    });
-    const searchInput = el.find('search-input').controller('searchInput');
+  it('displays search input in the sidebar', () => {
+    fakeStore.filterQuery.returns('test-query');
+    const wrapper = createTopBar();
+    assert.equal(wrapper.find(SearchInput).prop('query'), 'test-query');
+  });
+
+  it('updates search in the sidebar', () => {
```

Comment:
The description of this test is a little confusing...

Reply:
Agreed. It is out of date. I'll update it.